docker/jenkins/publish-build.sh

```diff
@@ -176,13 +176,76 @@ else
   base64_contents=$(echo "$md_contents" | base64 --wrap=0)
 fi
 
-payload="{\"message\":\"Add $flower build $version in $build\",\"content\":\"$base64_contents\",\"sha\":\"$sha256\"}"
-echo "Sending to Github: $payload"
+githubUrl="https://api.github.com/repos/rstudio/latest-builds/contents/content/rstudio/$flower/$build/$version_stem.md"
+curlOutFname="curl.out"
 
-curl \
-  -X PUT \
-  -H "Accept: application/vnd.github.v3+json" \
-  -H "Authorization: token $pat" \
-  "https://api.github.com/repos/rstudio/latest-builds/contents/content/rstudio/$flower/$build/$version_stem.md" \
-  -d "$payload"
+# Sha is only required in the payload if it's an update, removing the sha field gives a clearer
+# error message
```

Comment:
Is this because the dailies site calculates the sha itself?

Reply:
See above